src/SFML/Graphics/RenderTexture.cpp

```diff
@@ -40,19 +40,6 @@
 #include <SFML/Window/ContextSettingsInternal.h>
 
 
-////////////////////////////////////////////////////////////
-sfRenderTexture* sfRenderTexture_create(unsigned int width, unsigned int height, bool depthBuffer)
-{
-    sfRenderTexture* renderTexture = new sfRenderTexture;
-    renderTexture->This.create(width, height, depthBuffer);
-    renderTexture->Target = new sfTexture(const_cast<sf::Texture*>(&renderTexture->This.getTexture()));
-    renderTexture->DefaultView.This = renderTexture->This.getDefaultView();
-    renderTexture->CurrentView.This = renderTexture->This.getView();
-
-    return renderTexture;
-}
```

Comment:
Why was this removed?

Reply:
The function was deprecated according to RenderTexture.h

Reply:
Ah okay thanks